### PR TITLE
Add new LXC: Kubo for IPFS

### PR DESCRIPTION
**Pull request by ulmentflam**
## Description

The InterPlanetary File System (IPFS) is a set of composable, peer-to-peer protocols for addressing, routing, and transferring [content-addressed](https://docs.ipfs.tech/concepts/glossary/#content-addressing) data in a decentralized file system. 

### New Title to Match Application Name
Screenshot  Google Chrome-pve - Proxmox Virtual Environment 2024-06-26 at 21 16 54@2x

### Original PR Title
Screenshot  Google Chrome-pve - Proxmox Virtual Environment 2024-06-26 at 13 39 23@2x
Screenshot  Google Chrome-pve - Proxmox Virtual Environment 2024-06-26 at 13 40 03@2x

### Application
Screenshot  Google Chrome-Status  IPFS 2024-06-26 at 13 43 51@2x

I took an alternative approach to upgrading using the alternative install script because this package is lightweight. The install steps can be found here for reference https://github.com/ipfs/kubo. I also did a symlink to the /user/local/bin, I can change that if you prefer the files be copied, but the symlink is nice for upgrading.

## Type of change

- [x] New Script (Develop a new script or set of scripts that are fully functional and thoroughly tested)
- [x] I have performed a self-review of my code, adhering to established codebase patterns and conventions.
- [x] This change requires a documentation update

### CODE REVIEW

File: ct/ipfs.sh [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+ _/ // ____/ __/  ___/ /
+/___/_/   /_/    /____/
+EOF
+}
+header_info
+echo -e "Loading..."
+APP="IPFS"
+var_disk="4"
+var_cpu="2"
+var_ram="6144"
```
Comment (tteck) [inline ct/ipfs.sh]: Really? That's a lot of RAM

File: install/ipfs-install.sh [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+msg_info "Installing Dependencies"
+$STD apt-get install -y curl
+$STD apt-get install -y sudo
+$STD apt-get install -y mc
+$STD apt-get install -y gpg
+msg_ok "Installed Dependencies"
+
+msg_info "Installing IPFS"
+wget -q "$(curl -s "https://api.github.com/repos/ipfs/kubo/releases/latest" | grep "linux-amd64.tar.gz" | grep "browser_download_url" | head -n 1 | cut -d\" -f4)"
```
Comment (tteck) [inline install/ipfs-install.sh]: I prefer to get the version, then use a download link. This way you can use a conditional update. You'll see what I'm talking about in other scripts.

File: install/ipfs-install.sh [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+After=syslog.target network.target
+
+[Service]
+Type=simple
+ExecStart=/usr/local/bin/ipfs daemon
+Restart=on-failure
+[Install]
+WantedBy=multi-user.target
+EOF
+systemctl -q daemon-reload
```
Comment (tteck) [inline install/ipfs-install.sh]: not needed

File: ct/ipfs.sh (hunk 1 above)
Comment (ulmentflam) [reply ct/ipfs.sh]: I got conflicting reports on the minimum ram requirements, I believe it should run on 4096. I can update this.

File: install/ipfs-install.sh (hunk 2 above)
Comment (ulmentflam) [reply install/ipfs-install.sh]: I'm testing these changes. I updated to match the approach in `flaresolverr`

Comment (ulmentflam) [reply install/ipfs-install.sh]: Seems to be good

**Comment by tteck** [comment]
Looks like the actual application name is Kubo

**Comment by ulmentflam** [comment]
I can change that here shortly. I was initially debating what's more appropriate, but as long as the docs state it's for IPFS then it should be fine.

**Pull request merged**